### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by Swaagie**
## Summary

Update `README.md`, add badges and make it consistent with other Warehouse.ai repos

## Changelog

Done

## Test Plan

NA

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,36 +1,35 @@
...
+[![License](https://img.shields.io/npm/l/wrhs.svg?style=flat-square)](https://github.com/warehouseai/wrhs/blob/master/LICENSE)
+[![npm Downloads](https://img.shields.io/npm/dm/wrhs.svg?style=flat-square)](https://npmcharts.com/compare/wrhs?minimal=true)
+[![Build Status](https://travis-ci.org/warehouseai/wrhs.svg?branch=master)](https://travis-ci.org/warehouseai/wrhs)
+[![Dependencies](https://img.shields.io/david/warehouseai/wrhs.svg?style=flat-square)](https://github.com/warehouseai/wrhs/blob/master/package.json)
 
-
-[![License](https://img.shields.io/npm/l/wrhs.svg)](https://github.com/warehouseai/wrhs/blob/master/package.json)
+CLI for [Warehouse.ai]. CLI to manage build pipelines & asset deployments
+for your front-end apps powered by [Warehouse.ai]. A useful addition to
+the [Warehouse.ai UI][wrhs-ui].
```
Comment (msluther) [inline README.md]: >  A useful addition to the [Warehouse.ai UI][wrhs-ui].

I see the connection, but maybe reword as something more like `There is also a [web-based UI][wrhs-ui] available for [Warehouse.ai].`?

File: README.md [hunk 2]
```diff
@@ -41,14 +40,35 @@ USAGE
...
+
+## Commands
 <!-- commands -->
 * [`wrhs get:build PACKAGE ENV [LOCALE]`](#wrhs-getbuild-package-env-locale)
 * [`wrhs get:head PACKAGE ENV`](#wrhs-gethead-package-env)
 * [`wrhs get:status PACKAGE ENV`](#wrhs-getstatus-package-env)
 * [`wrhs help [COMMAND]`](#wrhs-help-command)
 
-## `wrhs get:build PACKAGE ENV [LOCALE]`
+### `wrhs get:build PACKAGE ENV [LOCALE]`
```
Comment (msluther) [inline README.md]: unfortunately I think these docs are all generated by oclif and this'll just get stomped next time its run.  I'm not sure if there's a way to customize it. :/

Comment (Swaagie) [reply README.md]: Good callout, I don't mind if it changes the header size. But I would be dissapointed if some of the other sections are gone. I'll double check. Perhaps it is as simple as adding an opt-out comment or removing the comment sections.

File: README.md [hunk 3]
```diff
@@ -1,47 +1,66 @@
-`wrhs` CLI for Warehouse.ai
-====
+# `wrhs`
 
-Flawless CLI management of build pipelines & asset deployments for your front-end apps powered by [Warehouse.ai].
```
Comment (fritzmonkey) [inline README.md]: 🙏 We can make mistakes again! 🙏

Comment (Swaagie) [reply README.md]: 😁 it read a little bit weird. Don't mind to bring it back but would need somewhat different wording
